src/components/TodoList.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+type Props = {
+  todos: Todo[],
+  removeTodo: (todoId: number) => void,
+  loadedTodoId: number[],
```

Comment:
Change the name of 'loadedTodoId' to 'loadedTodoIds' to better represent its purpose as an array of loaded todo IDs.